spec/api-remote-spec.js

```diff
@@ -552,6 +548,9 @@ describe('remote module', () => {
   })
 
   describe('remote function in renderer', () => {
+    let w = null
+
+    afterEach(() => closeWindow(w).then(() => { w = null }))
```

Comment:
combine the two `afterEach` blocks ?

Reply:
I'm not sure which I like better and I don't think it matters much so I'm going to go ahead and merge :)